### PR TITLE
fix/load current user location starting from CR

### LINKED ISSUE #520: Map component in pre-register lifebank load in my location
Location is pre-loaded when logging into the app.

- [ ] Test in other devices (mobile and other computers)

Own: @leisterfrancisco 
Sup: @steph222

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Resolves #520 

### What does this PR do?
Load current user location in the map section.

### How should this be tested?
- Make run
- Go to register lifebank
- Sroll down to map location

### CODE REVIEW

File: webapp/src/components/MapSelectLocation/index.js [hunk 1]
```diff
@@ -48,6 +48,16 @@ function MapSelectLocation({
       })
     )
 
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition((position) => {
+        // [position.coords.latitude, position.coords.longitude];
```
Comment (JustinCast) [inline webapp/src/components/MapSelectLocation/index.js:53]: It is necessary to maintain these commentaries?

File: webapp/src/components/MapSelectLocation/index.js [hunk 2]
```diff
@@ -48,6 +48,16 @@ function MapSelectLocation({
...
+        map.flyTo({
+          center: [position.coords.longitude, position.coords.latitude],
+          // essential: true // this animation is considered essential with respect to prefers-reduced-motion
```
Comment (JustinCast) [inline webapp/src/components/MapSelectLocation/index.js:56]: What about this?

**Review by JustinCast — APPROVED** [review]
LGTM

**Review by AngeloCG97 — APPROVED** [review]

**Review by Edgar097 — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**